src/core/plugins/oas3/components/operation-servers.jsx

```diff
@@ -27,6 +27,13 @@ export default class OperationServers extends React.Component {
     return this.props.setSelectedServer(server, `${path}:${method}`)
   }
 
+  updateServers = (server) => {
+    // FIXME: we should be keeping up with this in props/state upstream of us
```

Comment:
This is following the same pattern as the other methods in this file, thus the copied "FIXME" comment.